test/system/engine-and-editor-api.js

```diff
@@ -312,6 +311,10 @@ describe("Community product demo but through a running E&E instance", () => {
         const balanceBefore2 = await token.balanceOf(address2)
         log(`   Token balance before: ${formatEther(balanceBefore2)}`)
 
+        // Attempt to avoid "frozen" errors
+        await sleep(3000)
```

Comment:
@hpihkala ideally we shouldn't add this until we have an alternative failing test that captures the `error_proof` and `error_frozen` issues.

Reply:
👍